fn.go

```diff
@@ -187,7 +187,7 @@ func (f *Function) RunFunction(_ context.Context, req *fnv1.RunFunctionRequest)
 	}
 
 	// Initialize the requirements.
-	requirements := &fnv1.Requirements{ExtraResources: make(map[string]*fnv1.ResourceSelector)}
+	requirements := &fnv1.Requirements{Resources: make(map[string]*fnv1.ResourceSelector)}
```

Comment:
Switching from ExtraResources to Resources will break things
See https://github.com/crossplane/function-sdk-go/issues/219

Reply:
Yep, that's why I put it back in Draft.  We need to maintain the existing API to the user, which means adding the "extraResources" key into the template context with the same content as "requiredResources".  Not ideal, but here we are.